iothub/service/src/Common/Exceptions/DeviceMessageLockLostException.cs

```diff
@@ -7,9 +7,11 @@
 namespace Microsoft.Azure.Devices.Common.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to communicate with a device fails
-    /// because the lock token was lost (if the connection is lost and regained for example).
-    /// This timeout has the same effect as if the message was abandoned.
+    /// This exception is not directly returned by the service for ServiceClient operations. However, the following
+    /// status codes are converted to this exception:
+    /// <list>
+    /// <item>HttpStatusCode.PreconditionFailed</item>
```

Comment:
If it is a list of 1, shall we just mention the one error code? As in, no list?

Reply:
Yes we can do that. I thought there would be more, but it ended up being just one.